examples/waste_stats.py

```diff
@@ -47,6 +48,13 @@ def load_job_series(filename=None) -> pd.DataFrame:
     return df
 
 
+# to access series, you need prometheus access rights. This is doable only with `SARC_MODE=scraping` for the moment
```

Comment:
We should instead change the example to use `job.stored_statistics`. In fact, maybe the fetching should not be part of the `Job` object interface, because typically `sarc` client should be used without access to prometheus.

Reply:
Totally agree. This problem with clients needing access to scraping rights bothers me, too.
But this could be done in another PR more focused on examples (maybe [PR 86](https://github.com/mila-iqia/SARC/pull/86) ?)

Reply:
Bien d'accord, j'ai créé https://mila-iqia.atlassian.net/browse/SARC-244 pour pas qu'on perde ça de vu.